schemas/expconf/v0/experiment.json

```diff
@@ -71,11 +71,27 @@
         "entrypoint": {
             "type": [
                 "string",
+                "array",
                 "null"
             ],
             "checks": {
-                "entrypoint must be of the form \"module.submodule:ClassName\"": {
-                    "pattern": "^[a-zA-Z0-9_.]+:[a-zA-Z0-9_]+$"
+                "entrypoint must contain a trial class of the form \"module.submodule:ClassName\"": {
+                    "contains": {
+                        "pattern": "^[a-zA-Z0-9_.]+:[a-zA-Z0-9_]+$"
+                    }
+                }
+            },
+            "items": {
+                "type": "string"
+            },
+            "if": {
+                "type": "string"
+            },
+            "then": {
+                "checks": {
+                    "entrypoint must contain a trial class of the form \"module.submodule:ClassName\"": {
+                        "pattern": "(^| )[a-zA-Z0-9_.]+:[a-zA-Z0-9_]+.*$"
+                    }
```

Comment:
What if the user has an entrypoint like:
```yaml
entrypoint: ./train.py
```
Should that be valid?

Reply:
no. until we have the trainer API available, harness needs module:ClassName specification to load context and instantiate user code.